docs/api/model_selection/cross_validation.rst

```diff
@@ -30,13 +30,15 @@ In the following example we show how to visualize cross-validated scores for a c
     from yellowbrick.model_selection import CVScores
 
 
-    room = load_data("occupancy")
+    # Load the classification data set
+    data = load_data("occupancy")
```

Comment:
In our other code examples, we load the sample datasets into a variable named `data`, so I changed `room` here to `data`  to avoid any confusion.

Reply:
this is a great change!